btrfs.h

```diff
@@ -15,22 +15,28 @@
 */
 
 #include <stddef.h>
+#include <stdint.h>
+#include <string.h>
+#include <linux/version.h>
+#if LINUX_VERSION_CODE < KERNEL_VERSION(4,7,0)
+#error "Headers from kernel >= 4.7 are required on compilation time (not on run time)"
+#endif
 #include <linux/magic.h>
```

Comment:
I think it actually might be better to move this into `btrfs.go` so it is clear where it comes from in that file/where it is used. The magic numbers are not really the same thing as the Btrfs API.

Reply:
done